tsconfig.json

```diff
@@ -1,7 +1,7 @@
 {
   "compilerOptions": {
     "jsx": "preserve",
-    "module": "system",
+    "module": "amd",
```

Comment:
why amd ?

Reply:
because - Option '--resolveJsonModule' can only be specified when module code generation is 'commonjs', 'amd', 'es2015' or 'esNext'.

Reply:
use commonjs then.

Reply:
Only 'amd' and 'system' modules are supported alongside --outFile.

Reply:
what is amd, I havent used and I think is not appropriate. Recently I am using esnext.

Reply:
done